packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -357,7 +357,7 @@ export const DevServerMixin = {
     _getWebpackAsset(req, compilerName, fileName) {
         if (req.app.__webpackReady()) {
             const outputFileSystem = req.app.__devMiddleware.context.outputFileSystem
-            const jsonWebpackStats = req.app.__devMiddleware.context.stats.toJson()
+            const jsonWebpackStats = req.app.__devMiddleware.context.stats.toJson({preset: 'none', outputPath: true})
```

Comment:
What does this change do? How does it impact people who aren't seeing excessive slowdowns? Is there anything we can test for to validate these changes work as expected? (Not blocking, just questions.)

Reply:
@wjhsf what `stats.toJson` does is essentially getting the stats data in JSON format, and you have the option to selectively query the piece of data you want/need. If we don't pass in the option, by default, you'll get _all_ of the data. Depending on the project, you may get a lot of data.

And so the conversion process to JSON may run slowly if there's a lot of data to generate. 

In our function, we only need 1 piece of data `outputPath`. Hence, we now pass in the option `{preset: 'none', outputPath: true}`:
1. start off with `none` which is no data at all
2. and then add the `outputPath`

That option will not affect people who are not seeing the slowdowns. But will be beneficial to projects who have huge stats data.

Reply:
I'm curious about the performance gain for this change for the OOTB retail react app. How much time are we saving in ballpark?

Reply:
btw, it would be nice to add a comment to explain the "why"